hs_swat_modelinstance/tests/test_swat_modelinstance_metadata.py

```diff
@@ -0,0 +1,824 @@
+                                                                                 s_objs[1]: s_objs[1]},
+                                                          other_objectives=o_objs)
+        self.resSWATModelInstance.metadata.create_element('SimulationType',
+                                                          simulation_type_name='Normal Simulation')
+        self.resSWATModelInstance.metadata.create_element('ModelMethod',
+                                                          runoffCalculationMethod='aaa',
+                                                          flowRoutingMethod='bbb',
+                                                          petEstimationMethod='ccc')
+        s_params = ["Crop rotation", "Tillage operation"]
+        o_params = "spongebob"
+        self.resSWATModelInstance.metadata.create_element('ModelParameter',
+                                                          model_parameters={s_params[0]: s_params[0],
+                                                                            s_params[1]: s_params[1]},
+                                                          other_parameters=o_params)
+
+        self.resSWATModelInstance.metadata.create_element('ModelInput',
+                                                          warmupPeriodValue='a',
+                                                          rainfallTimeStepType='Daily',
+                                                          rainfallTimeStepValue='c',
+                                                          routingTimeStepType='Daily',
+                                                          routingTimeStepValue='e',
+                                                          simulationTimeStepType='Hourly',
+                                                          simulationTimeStepValue='g',
+                                                          watershedArea='h',
+                                                          numberOfSubbasins='i',
+                                                          numberOfHRUs='j',
+                                                          demResolution='k',
+                                                          demSourceName='l',
+                                                          demSourceURL='m',
+                                                          landUseDataSourceName='n',
+                                                          landUseDataSourceURL='o',
+                                                          soilDataSourceName='p',
+                                                          soilDataSourceURL='q',
+                                                          )
+
+        # test if xml from get_xml() is well formed
+        ET.fromstring(self.resSWATModelInstance.metadata.get_xml())  # todo check actual xml content
+
+    def test_metadata_on_content_file_delete(self):
+        # upload files

```

Comment:
Add comment here what you expect to happen to metadata on content file deletion.